zipline/pipeline/factors/basic.py

```diff
@@ -54,6 +54,30 @@ def compute(self, today, assets, out, close):
         out[:] = (close[-1] - close[0]) / close[0]
 
 
+class PercentChange(SingleInputMixin, CustomFactor):
+    """
+    Calculates the percent change over the given window_length.
+    Note: Uses formula "new - old / abs(old)" for negative input values
```

Comment:
Minor point, but this sentence could be read as saying that this is only the formula for negative input values (leaving it unclear what the formula is otherwise). I would probably just put something like. "Percent change is calculated as ``(new - old) / abs(old)``

Reply:
Regarding the "Note:" qualifier on this, we generally follow https://numpydoc.readthedocs.io/en/latest/format.html for our docstring convention, which among other things allows us to auto-generate relatively nice-looking documentation from these docstrings using [Sphinx](https://www.sphinx-doc.org/en/master/). If you want to add "notes", to a docstring, the usual convention would be to add a section that looks like:

```
Notes
-----
Your content here.
```

Normally we would also document with a `Parameters` section (you'll see lots of examples of those around elsewhere in the codebase), but we don't do so for pipeline expressions b/c they have a bit of a weird construction model (related to all the `__new__`s floating around in the base classes).